anda/apps/rp-appset/rp-appset.spec

```diff
@@ -0,0 +1,60 @@
+%global commit 5b4b8f65c3d2795a61e765a01e07af9bfe3d1990
+%global commit_date 20250501
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+
+Name:           appset
+Version:        0~%commit_date.git~%shortcommit
+Release:        1%?dist
+Summary:        Application for customisation of appearance of Raspberry Pi Desktop
+License:        BSD-3-Clause
+URL:            https://github.com/raspberrypi-ui/appset
+Source0:        %url/archive/%commit.tar.gz
+Packager:       Owen Zimmerman <owen@fyralabs.com>
+
+BuildRequires:  meson
+BuildRequires:  ninja-build
+BuildRequires:  gtk3-devel
+BuildRequires:  libxml2-devel
+BuildRequires:  intltool
+BuildRequires:  gcc
+
+Requires:       libxml2
+Requires:       gtk3
+
+Provides:       pipanel
+Provides:       rp-appset
+
+%description
+%summary.
+
+%prep
+%autosetup -n appset-%commit
+
+%build
+%meson
+%meson_build
+
+%install
+%meson_install
+
+%files
+%doc README
+%license debian/copyright
+%{_bindir}/pipanel
+%{_datadir}/applications/pipanel.desktop
+%{_datadir}/pipanel/ui
+%{_datadir}/locale/de/LC_MESSAGES
+%{_datadir}/locale/en_GB/LC_MESSAGES
```

Comment:
you can't own this dir, otherwise this conflicts with every other packages that install their mo files here